scala-package/core/src/test/scala/org/apache/mxnet/NDArraySuite.scala

```diff
@@ -335,7 +335,7 @@ class NDArraySuite extends FunSuite with BeforeAndAfterAll with Matchers {
   }
 
   test("arange") {
-    for (i <- 0 until 5) {
+    for (i <- 0 until 10000) {
```

Comment:
Instead of increasing the number of iterations, could you maybe add a test case that specifically hits the behaviour you're trying to circumvent here?

Reply:
Sure. That's definitely a better solution.

Reply:
Awesome, thanks!

Reply:
@lanking520 @nswamy could you give me a hand here and review the scala code?